tests/integration/test_utils/env.rs

```diff
@@ -0,0 +1,25 @@
+//! Utilities for setting environment variables in integration tests.
+
+use std::borrow::Cow;
+
+/// Set the environment variables, which should be set for all integration tests,
+/// using the provided setter function.
+/// The setter function takes as parameters the environment variable name, and the
+/// value to set it to, in that order.
+pub fn set(setter: impl Fn(&'static str, Cow<'static, str>)) {
```

Comment:
This should be `FnMut`, but you already caught that in your other PR.

Reply:
Fixed this in this PR